.github/workflows/update-vcluster-docs.yaml

```diff
@@ -0,0 +1,109 @@
+name: Update partials in documentation
```

Comment:
Can we add `actor` name triggering the event? It's optional, but helpful when looking at the `Actions` tab.

```yaml
run-name: ${{ github.actor }} is updating partials
```

Reply:
that's a good idea, I wasn't aware of `github.actor` :+1